examples/frameworks/strands_demo/bedrock_agentcore/README.md

```
@@ -21,53 +21,117 @@ A comprehensive guide for deploying NVIDIA NeMo Agent toolkit with Strands on AW
+- **CloudWatch Console** - To enable Transaction Search and view logs and traces (see [Appendix 2](#appendix-2-turning-on-opentelemetry-support-in-cloudwatch))
+- **Secrets Manager Console** - To manage the NVIDIA API credentials secret
+
+> NOTE: Details instructions for setting up IAM permissions in the AWS console are available in Appendix 1
+
+### Additional Requirements
+
+- **NVIDIA API Key** - Obtain from [NVIDIA NGC](https://ngc.nvidia.com/) or [build.NVIDIA](https://build.nvidia.com). This will be stored in AWS Secrets Manager during setup.
+
+## Step 1: Setup NeMo Agent Toolkit Environment
+
+Follow the official NeMo Agent toolkit installation guide: [here](https://docs.nvidia.com/nemo/agent-toolkit/latest/quick-start/installing.html)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)